embed.fnc

```diff
@@ -3716,7 +3716,7 @@ Cp	|bool	|utf8_to_bytes_ |NN U8 **s_ptr				\
 Admp	|bool	|utf8_to_bytes_new_pv					\
 				|NN U8 const **s_ptr			\
 				|NN STRLEN *lenp			\
-				|NN U8 *free_me
+				|NN const U8 *free_me
```

Comment:
I may have missed this, since it's purely a documentation issue, but `free_me` isn't a pointer to U8, it certainly isn't a pointer to const U8.

`free_me` would ideally be declared as `void **` but the existing users are passing in a `U8 **` which makes that more reasonable here.

And it shouldn't be const in any way, free() requires a non-const pointer, it's only a bug that Safefree() casts const away.  Your modification to the macro below also casts const away.

Using inline functions instead of macros here would have made the type mismatches a lot more obvious.